### ISSUE TITLE
All currencies instead of top 100

### ISSUE THREAD

**Issue by arno-github** [Nexuist/CoinMarketCapAPI#7]
Thanks for your work, first :-)
I'd like to use your piece of code on my VPS, but I'm not a coder, so I need a more detailed README or a INSTALL file, please !
The aim would be to be able to have all cryptocurrencies prices, not just the top 100... Would you please add this option, or create an INSTALL file ;-) ?

**Comment by Nexuist** [comment]
Installation instructions have been included in the README with 2.0, it's now very simple :)

As to having all prices, I'm afraid that's outside of the scope of this project. The goal is top 100 only.

**Comment by altsheets** [comment]
@Nexuist and @githare I have found a simple hack which can help to achieve this 

```
cp -r Nexuist_CoinMarketCapAPI  Nexuist_CoinMarketCapAPI_2
cd Nexuist_CoinMarketCapAPI_2/
nano lib/CoinCrawler.js 
```
change 'http://coinmarketcap.com/' to 'http://coinmarketcap.com/2'

then increment the port number so that page 1 & page 2 do not collide:
```
nano main.js 
```
change 8080 to 8081

now install and run:
```
rm node_modules -rf
npm install
npm start
```

then you get page 2 on http://try.your.ip.address:8081

To not miss any coins (which move from page 1 to page 2 or vice versa), it is important that both services 8080 and 8081 are started at the same time.
